_posts/2023-05-15-oss-na.md

```diff
@@ -10,14 +10,14 @@ show_pagenav: false
 Last week I had the pleasure to attend [Open Source Summit North America
 2023](https://events.linuxfoundation.org/open-source-summit-north-america/) in
 Vancouver. A quick hop across the [Strait of
-Georgia](https://en.wikipedia.org/wiki/Strait_of_Georgia) got my right into the
-event and in the midst of my peers of open source developers, advocates, and
+Georgia](https://en.wikipedia.org/wiki/Strait_of_Georgia) got me right into the
+event, and in the midst of my peers of open source developers, advocates, and
 enthusiasts.
 
 <!--more-->
 
 A highlight of the event for me was catching up with many existing and new
-friends from the open source communities. It was also inspiring to learn details
+friends from the open source communities. It was inspiring to learn details
 about the success of open source projects and communities, including
```

Comment:
```suggestion
about the success of open source projects, including
```
This ends up being kind of redundant, the previous sentence ends with the word "communities"